tests/mocha/server/RoundTest.js

```diff
@@ -41,9 +49,22 @@ MochaWeb.testOnly(function() {
       chai.expect(open.isClosed()).to.be.false;
       chai.expect(closed.isClosed()).to.be.true;
     });
+
+    it('displayTitle()', function() {
+      chai.expect(makeRound({eventCode: 'skewb', nthRound: 1, totalRounds: 2}).displayTitle()).to.equal('Skewb: Round 1 of 2');
+      chai.expect(makeRound({eventCode: 'skewb', nthRound: 1, totalRounds: 1}).displayTitle()).to.equal('Skewb: Single Round');
+    });
+
+    it('isLast()', function() {
+      chai.expect(makeRound({nthRound: 1, totalRounds: 2}).isLast()).to.be.false;
+      chai.expect(makeRound({nthRound: 2, totalRounds: 2}).isLast()).to.be.true;
+    });
   });
 
   function makeRound(properties) {
-    return Rounds.findOne(Rounds.insert(_.extend({ competitionId: 'fake'}, properties)));
+    return Rounds.findOne(Rounds.insert(_.extend({ competitionId: 'fake', nthRound: 1, totalRounds: 1 }, properties)));
```

Comment:
I thought about it, and it feels a little premature.

It's absolutely the right thing but I'd prefer to have some more tests to inform the design and make it worth the effort.